client/webserver/site/src/js/markets.ts

```diff
@@ -802,6 +799,21 @@ export default class MarketsPage extends BasePage {
       Doc.bind(bttn, 'click', () => this.candleDurationSelected(dur))
       page.durBttnBox.appendChild(bttn)
     }
+
+    // Check if we already have the fiveMinBin candles cache, if not, request
+    // for it first since we use it to determine the 24hour high/low in
+    // this.setHighLow().
+    const lastCandleDur = State.fetchLocal(State.lastCandleDurationLK)
+    const cache = this.market?.candleCaches[fiveMinBinKey]
+    if (!cache && lastCandleDur !== fiveMinBinKey) {
+      this.candleDur = fiveMinBinKey
```

Comment:
OK we super need this on 0.6, and to be changed to the 1hr bin key.
Can we get this updated today?

Reply:
Are we going to use `spot.high24` and `spot.low24` now?

Reply:
It's available from all dex hosts I'm aware of.
What happens if it's not set?  Can we detect if it's zero and do something differently without big changes?

Reply:
> Can we detect if it's zero and do something differently without big changes?

Yeah. Okay.

Reply:
Or to be safe we can keep the `lowRate`/`highRate` loop and just make the default candles 1hr.  I don't follow the problem exactly.  Maybe @buck54321 has a simple way to change the default while keeping the current high/low logic the way it is, not using the spot note, which could cause a regression.

Reply:
Like maybe after the initial `loadCandles` using `fiveMinBinKey` like it currently does, we just do a `this.candleDurationSelected('1hr')` right after it.  It's kinda dumb, but it might be fine.

Reply:
```diff
diff --git a/client/webserver/site/src/js/markets.ts b/client/webserver/site/src/js/markets.ts
index 2e7451bd4..44664421e 100644
--- a/client/webserver/site/src/js/markets.ts
+++ b/client/webserver/site/src/js/markets.ts
@@ -85,6 +85,7 @@ const buyBtnClass = 'buygreen-bg'
 const sellBtnClass = 'sellred-bg'
 
 const fiveMinBinKey = '5m'
+const oneHrBinKey = '1h'
 
 const percentFormatter = new Intl.NumberFormat(document.documentElement.lang, {
   minimumFractionDigits: 1,
@@ -874,7 +875,8 @@ export default class MarketsPage extends BasePage {
 
     // candlesticks
     this.candleDur = fiveMinBinKey
-    this.loadCandles()
+    this.loadCandles() // gotta do this with fiveMinBinKey first to get high/low
+    this.candleDurationSelected(oneHrBinKey)
 
     this.setLoaderMsgVisibility()
     this.setRegistrationStatusVisibility()
```

That way we don't have to worry about an old server lacking high/low in the spot note.